src/core/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,166 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { useTheme } from '../utils/hooks/useTheme';
+import { CommonProps } from '../utils/props';
+import SvgChevronRight from '@itwin/itwinui-icons-react/cjs/icons/ChevronRight';
+import '@itwin/itwinui-css/css/breadcrumbs.css';
+import { useResizeObserver } from '../utils/hooks/useResizeObserver';
+import { useMergedRefs } from '../utils/hooks/useMergedRefs';
+
+export type BreadcrumbsProps = {
+  /**
+   * Index of the currently active breadcrumb.
+   * Defaults to the index of the last breadcrumb item.
+   */
+  currentIndex?: number;
+  /**
+   * Breadcrumb items.
+   */
+  children: JSX.Element[];
```

Comment:
It looks like when its type is `JSX.Element[]`, you don't need to use `React.Children`. Or you can change to `ReactNode`.

Reply:
Changed to ReactNodeArray